### DISCUSSION TITLE
Block to modal

### DISCUSSION THREAD

**Discussion by JuniorVezintana** [GrapesJS/grapesjs#5235] [General]
How to create a block that opens a modal, so I can add variations of a block, inside it, so I have a single block in the column with the name "Button" and it opens a modal, with several different button blocks...

**Comment by artf** [comment]
Hi @JuniorVezintana please use the search functionality in Discussions, there are plenty of similar questions

**Comment by JuniorVezintana** [comment]
I didn't find any that help me.
